CONTRIBUTING.md

```diff
@@ -16,6 +16,11 @@ In order to set up this project and start contributing follow this step by step
     ```sh
     $ git clone git@github.com:<your-username>/webdriverio.git
     ```
+* On Windows, you might need to set git config `core.symlinks` to `true` as we currently has some symbolic links for type definition commited to our repo.
```

Comment:
Should we change the wording towards something like "Ensure you have this config set" - because we know this setting is necessary, no? This, as is, sounds more like you might not need to set it.

Reply:
I added "might" in the sense that we'll probably remove this symlinks in the future, but you're right, too. I don't really see why we need this symlinks. We can specify typings directly in package.json, as pointed out on https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package. Do you happen to know why we need the symlinks?